docs/tutorials/create-a-server.md

```diff
@@ -0,0 +1,108 @@
+# Creating a server
+
+This tutorial covers the process of creating a server using the Hetzner Cloud CLI. It includes creating an SSH Key, 
+uploading it, creating a Server and then connecting to it via SSH.
+
+## Prerequisites
+
+- A functioning installation of the [hcloud CLI](setup-hcloud-cli.md), with a valid active context.
+
+## 1. Create an SSH Key
+
+### 1.1 Generate an SSH Key
+
+While an SSH key is not strictly required to create a server, it is highly recommended for secure access.
+If you don't have an SSH key yet, you can generate one using the following command:
+
+```bash
+ssh-keygen -t ed25519 -f ~/.ssh/hcloud
+```
+
+Your private key will now be located at `~/.ssh/hcloud`. **Do not share your private key with anyone!**
+
+### 1.2 Upload the SSH Key
+
+You can upload your SSH key to Hetzner Cloud using the following command:
+
+```bash
+hcloud ssh-key create --name my-ssh-key --public-key-from-file ~/.ssh/hcloud.pub
+```
+
+**Pro Tip**: You can set this SSH key as the default SSH key for your context using the following command:
```

Comment:
You can use alert block to put some colors in the docs: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts